app/code/Magento/QuoteGraphQl/Model/Cart/AddSimpleProductToCart.php

```diff
@@ -87,7 +87,8 @@ public function execute(Quote $cart, array $cartItemData): void
         }
 
         if (is_string($result)) {
-            throw new GraphQlInputException(__($result));
+            $uniqueMessqges = array_unique(explode("\n", $result));
```

Comment:
Generally, need to investigate why do we have not unique messages... looks like a bug of Cart

Reply:
Changes were reverted. 
FIx  was applied inbase level.